MACSio.spec

```
@@ -45,32 +58,74 @@ We hope MACSio helps to put the MAX in scalable I/O performance ;)
 
 The name "MACSio" is pronounced max-eee-oh.
 
+
+%if %{with_mpich}
+%package mpich
+Summary: MACSio for MPICH
+BuildRequires: hdf5-mpich-devel%{?_isa}
+BuildRequires: mpich-devel%{?_isa}
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description mpich
+MACSio for MPICH
+%endif
+
+%if %{with_openmpi3}
+%package openmpi3
+Summary: MACSio for OpenMPI 3
+BuildRequires: hdf5-openmpi3-devel%{?_isa}
+BuildRequires: openmpi3-devel%{?_isa}
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description openmpi3
+MACSio for OpenMPI 3
+%endif
+
 %prep
 %setup -q
 
 %build
 %if (0%{?suse_version} >= 1500)
-  module load gnu-mpich
-%else
-  module load mpi/mpich-%{_arch}
+  sed -i -e s/H5pubconf.h/H5pubconf-64.h/ plugins/macsio_hdf5.c
 %endif
-cmake -DCMAKE_INSTALL_PREFIX=%{_bindir} \
-    -DWITH_JSON-CWX_PREFIX=/usr \
+for mpi in %{?mpi_list}
+do
+  mkdir $mpi
+  pushd $mpi
+  %module_load $mpi
+  cmake -DCMAKE_INSTALL_PREFIX=%{_bindir}/$mpi \
```

Isn't the typical mulit-mpi-variant model to install things into `%{_libdir}/$mpi/{bin,lib,etc.}`?  So:
```suggestion
  cmake -DCMAKE_INSTALL_PREFIX=%{_libdir}/$mpi \
```

---

Done